### DISCUSSION TITLE
Is it possible to set an Origin claim in the payload?

### DISCUSSION THREAD

**Discussion by robertbarnes93** [panva/jose#440] [Q&A] [answered]
I am using Apple's MapKit and noticed that the [documentation suggests adding `origin` to the payload](https://developer.apple.com/documentation/mapkitjs/creating_and_using_tokens_with_mapkit_js) to whitelist domains. I tried using `setAudience` but I believe it's for a different purpose. Is this something that `jose` can accommodate?

**Comment by panva** [comment, marked as answer]
You can set any jwt claim you want. Please have a look at the JWT specific API, claim set is an argument in the constructor.

**Reply by robertbarnes93** [reply]
Thanks for pointing me in the right direction. I was able to get it to work by using `jose.SignJWT({'origin': 'localhost'})`. Appreciate the help!
